dashboard/test/models/experiment_test.rb

```diff
@@ -1,22 +1,22 @@
 require 'test_helper'
 
 class ExperimentTest < ActiveSupport::TestCase
-  setup do
-    Experiment.stubs(:should_cache?).returns false
-  end
-
   test "no experiments" do
     assert_empty Experiment.get_all_enabled(user: create(:user), section: create(:section))
   end
 
   test "user based experiment at 0 percent is not enabled" do
-    create :user_based_experiment, percentage: 0
-    assert_empty Experiment.get_all_enabled(user: create(:user))
+    experiment = create :user_based_experiment, percentage: 0
+    user = create :user
```

Comment:
Can the creation of a user be lifted to `setup_all`? Also the creation of a section?